modules/acid/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
+### Fixed
+- Color Picker doesn't handle hex color prefixed with #.
```

Comment:
What was the JIRA code for this?

Reply:
I used the commit message  `TINY-9457: Color Picker doesn't handle hex color prefixed with #`